pom.xml

```diff
@@ -252,7 +249,7 @@
         <version.org.wildfly.build-tools>1.2.9.Final</version.org.wildfly.build-tools>
         <version.org.wildfly.cdi-api-bridge>1.0.1.Final</version.org.wildfly.cdi-api-bridge>
         <version.org.wildfly.checkstyle-config>1.0.5.Final</version.org.wildfly.checkstyle-config>
-        <version.org.wildfly.core>4.0.0.Final</version.org.wildfly.core>
+        <version.org.wildfly.core>5.0.0.Alpha1-SNAPSHOT</version.org.wildfly.core>
```

Comment:
We can't use snapshot deps for Wildfly. If you need a Wildfly core release just leave a comment on the PR and we will hold off merging until the release has been done.

Reply:
sry, relict of working version, fixed